scripts/operations/node_management/ncn-ims-image-upload.sh

```diff
@@ -72,13 +72,18 @@ cray artifacts create boot-images "$IMS_IMAGE_ID/rootfs" "$IMS_ROOTFS_FILENAME"
 cray artifacts create boot-images "$IMS_IMAGE_ID/kernel" "$IMS_KERNEL_FILENAME" > /dev/null
 cray artifacts create boot-images "$IMS_IMAGE_ID/initrd" "$IMS_INITRD_FILENAME" > /dev/null
 
+ROOTFS_ETAG=$( cray artifacts describe boot-images ${IMS_IMAGE_ID}/rootfs --format json | jq -r .artifact.ETag  | tr -d '"' )
```

Comment:
Generally I would recommend unsetting the `CRAY_FORMAT` environment in scripts that use `--format json` because `CRAY_FORMAT` at one point in time had precedence over the command line, so could break things.